Fixed comparable elements in line 313

BradeeL opened the pull request:
Originally said 1 and 3 were not comparable because 1<=3 was false and 3<=1 was false. But 1<=3 is true, so the elements are comparable.

BradeeL committed:
Fixed comparable elements in line 313 

Originally said 1 and 3 were not comparable because 1<=3 was false and 3<=1 was false. But 1<=3 is true, so the elements are comparable.

BennyWestsyde reviewed (changes requested):
Another great catch. Would you please make the same edits to the Relations.md file too?

BradeeL committed:
Updated relations.md

Changed relations.md to reflect changes made in review guide earlier. Also, added a new example of incomparable elements.

BradeeL commented:
Relations.md updated. I also added a new example of incomparable elements.

BennyWestsyde reviewed (approved):
This is fantastic. Great job, and thank you for your participation!
